examples/step-47/doc/results.dox

```diff
@@ -211,3 +211,81 @@ make sense:
   addition should not be overly difficult using, for example, the
   FEInterfaceValues class combined with MeshWorker::mesh_loop() in the
   same spirit as we used for the assembly of the linear system.
+
+
+  <h3> Derivation for the simply supported plates </h3>
+
+  Similar to the “clamped” boundary condition addressed in the implementation,
+  we will derive the $C^0$ IP finite element scheme for the simply supported plates:
+  @f{align*}{
+    \Delta^2 u(\mathbf x) &= f(\mathbf x)
+    \qquad \qquad &&\forall \mathbf x \in \Omega,
+    u(\mathbf x) &= g(\mathbf x) \qquad \qquad
+    &&\forall \mathbf x \in \partial\Omega, \\
+    \Delta u(\mathbf x) &= h(\mathbf x) \qquad \qquad
+    &&\forall \mathbf x \in \partial\Omega.
+  @f}
+  We multiply the biharmonic equation by the test function $v_h$ and integrate over $\Omega$ and get:
+  @f{align*}{
+    \int_K v_h (\Delta^2 u_h)
+     &= \int_K (D^2 v_h) : (D^2 u_h)
+       + \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}}
+       -\int_{\partial K} (\nabla v_h) \cdot (\frac{\partial \nabla u_h}{\partial \mathbf{n}}).
+  @f}
+
+  Summing up over all cells $K \in  \mathbb{T}$,
+  @f{align*}{
+  \sum_{K \in \mathbb{T}} \int_{\partial K} v_h \frac{\partial (\Delta u_h)}{\partial \mathbf{n}} = 0,
```

Comment:
Is it easy to see that this is so? It's not obvious to me right away, but maybe I'm just dense. In any case, maybe explain this in one sentence.

Reply:
Is that because on the boundary of domain, v = 0, so no contribution, on the interior faces, normal directions of \Delta u are pointing at opposite directions, so contributions from 2 cells canceled?

Reply:
@bangerth I'm not sure whether my understanding explained above is correct. Could you please help me with it? 
I will continue editing other parts during the break.

Reply:
I'll give it another read when you've addressed all of the other comments here. Just let me know when you're at that point!

Reply:
Ok! Thanks!